frontend/dev-mode/src/views/PromptsView.vue

```diff
@@ -63,7 +63,7 @@ const {
   runCompletion,
   stop,
   uploadImage
-} = useLlama(parameters.value)
+} = useLlama({ ...parameters.value, slot_id: -1 })
```

Comment:
this hard `-1` number is because the first time, it must be -1 but then should just use any value that came back from the api response.